src/components/VmConsoles/SerialConsoleConnector.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
```

Comment:
can you rename VmConsoles to VmConsole (singular) ? We are using singular for also for Wizard etc..

Reply:
I can, it's nothing I would fight for. But the component wraps access to multiple consoles. I thing the plural express it's meaning better in this case. 
The `Wizard` is just a folder - container for other (unrelated) components. The `VmConsoles` is the component itself focused on managing multiple sub-components.

Reply:
you are right. Lets keep it as it is.